setup.py

```diff
@@ -26,6 +26,28 @@ def package_data(pkg, root_list):
     zip_safe=False,
     packages=find_packages(),
     include_package_data=True,
+    classifiers=[
+        'Development Status :: 5 - Production/Stable',
+        "Intended Audience :: Developers",
+        "License :: Affero GNU General Public License v3 (GPLv3)",
+        'Operating System :: OS Independent',
+        "Natural Language :: English",
+        'Environment :: Web Environment',
+        'Framework :: Django',
+        'Framework :: Django :: 2.2',
```

Comment:
If this change won't be compatible with django 2.2, shouldn't we remove this?

Reply:
I would update it once I will finalize the PR.